tests/uintptr_t.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
+import dace
+import ctypes
+import numpy as np
+
+
+def test_uintptr_t():
```

Comment:
Test is irrelevant, please try using it in a dace program so that we see the full code generation pipeline working. See tests/numpy for some examples you can add the type to